website/content/en/docs/setup/installation/operating-systems/nixos.md

```diff
@@ -11,46 +11,116 @@ page covers installing and managing Vector on NixOS.
 Nixpkgs has a [community maintained module][nixpkg-vector] for Vector, the
 options for which may be viewed on the [NixOS Search][nixos-search].
 
-This can be used to deploy and configure Vector on a NixOS system.
+This can be used to deploy and configure Vector on a NixOS system, the module
+will also verify that the Vector configuration is valid before enabling any
+changes.
```

Comment:
```suggestion
This can be used to deploy and configure Vector on a NixOS system. The module also verifies that the Vector configuration is valid before enabling any
changes.
```